mobsf/DynamicAnalyzer/views/android/environment.py

```diff
@@ -509,7 +509,11 @@ def launch_n_capture(self, package, activity, outfile):
                           'start',
                           '-n',
                           package + '/' + activity], True)
-        self.wait(settings.ACTIVITY_TESTER_SLEEP)
+        if not getattr(settings, 'ACTIVITY_TESTER_SLEEP'):
```

Comment:
@ajinabraham Are you thinking of an option where a user pulled the latest version, but the settings.py wasn't updated? 
I mean, if he updated the the project, both the activity tester + the settings should be updated, or am I missing something

Reply:
Not his `settings.py` but`~/.MobSF/config.py` will not be updated which Django uses and will result in an exception.